public/app/features/alerting/unified/utils/misc.ts

```diff
@@ -153,8 +153,22 @@ export function makeDashboardLink(dashboardUID: string): string {
   return createUrl(`/d/${encodeURIComponent(dashboardUID)}`);
 }
 
-export function makePanelLink(dashboardUID: string, panelId: string): string {
-  return createUrl(`/d/${encodeURIComponent(dashboardUID)}`, { viewPanel: panelId });
+type PanelLinkParams = {
+  viewPanel?: string;
+  editPanel?: string;
+  tab?: 'alert' | 'transform' | 'query';
+};
+export function makePanelLink(dashboardUID: string, panelId: string, queryParams: PanelLinkParams = {}): string {
+  const panelParams = new URLSearchParams({
+    viewPanel: panelId,
```

Comment:
nit: Doesn't `...queryParams` do the job?

Reply:
Oh that's a good point – I think I refactored this and forgot to simplify this part :D